adview-manager/src/lib.rs

```diff
@@ -231,7 +224,7 @@ pub fn get_unit_html_with_events(
             .iter()
             .map(|validator| {
                 let fetch_url = format!(
-                    "{}/channel/{}/events?pubAddr={}",
+                    "{}/campaign/{}/events?pubAddr={}",
```

Comment:
So what should we do with the `pubAddr`?

Reply:
It's not used in the validator but it's used for analytics/cloudflare